integration/integration.go

```diff
@@ -107,6 +162,11 @@ func (i *Integration) MarshalJSON() (output []byte, err error) {
 	return
 }
 
+// Logger returns the integration logger instance.
```

Comment:
I thought the log was provided by the user to the integration, for internal logging purposes. Do we want to expose that? In other words: when the user would need to log on behalf of the integration?

Reply:
I added a getter in purpose, because we are returning errors when calling: NewIntegration, Entity, NewMetricSet, Publish... and the user would probably log them on a new logger or discard them.

Reply:
To simplify the usage, there is a default logger. The user could like to get it and use it later. What do you think?

Reply:
That's the current usage indeed.